tutorials/colors/colormaps.py

```diff
@@ -349,7 +351,7 @@ def plot_color_gradients(cmap_category, cmap_list, nrows):
 # overlaid, labeled contours could help differentiate between one side of the
 # colormap vs. the other since color cannot be used once a plot is printed to
 # grayscale. Many of the Qualitative and Miscellaneous colormaps, such as
-# Accent, hsv, and jet, change from darker to lighter and back to darker gray
+# Accent, hsv, jet and turbo, change from darker to lighter and back to darker gray
```

Comment:
We have a 79-character limit, so this won't pass our format requirement test

Reply:
Fixed it up, though some lines above are at 80 chars, just fyi... (should I fix those too?)

Reply:
Currently this file is excluded from the 80 char limit. https://github.com/matplotlib/matplotlib/blob/a813b92e3947a42c410213cf846b5c1788d061b0/.flake8#L53

This means you can either leave it as it is or fix all of them and remove the entry in the `.flake8` file. (One should be aware that the more changes are made at different points, the higher the chances one will later on need to rebase; this might speak in favour of doing those line-length changes in a separate PR)